### DISCUSSION TITLE
pass all receiving emails to HTTP endpoint without creating  routes

### DISCUSSION THREAD

**Discussion by thilina15** [postalserver/postal#2141] [Help with using Postal] [answered]
in postal web panel, we have to configure routes for each email address to receive the emails. 
when we have large number of users with different email address, we can't create routes for each and every emails in the system.
is there any option to do this.  forward all the receiving emails into a HTTP endpoint without creating routes for each email address. 
thank you.

**Comment by willpower232** [comment, marked as answer]
Based on #57 you should be able to enter `*` as the route and get all your email forwarded.

**Reply by thilina15** [reply]
@willpower232 thank you for the quick reply. and it worked..
